extensions/shadcn-ui/src/search-components.tsx

```diff
@@ -0,0 +1,88 @@
+import { ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useFetch, Response } from "@raycast/utils";
+
+export default function Command() {
+  const { data, isLoading } = useFetch("https://ui.shadcn.com/api/components", {
```

Comment:
maybe you can check if a `useCachedPromise` could fit the use case

Reply:
done 19959d3f312be4d66dbada5c847922d7b95327ab 😉